### ISSUE TITLE
Can I run multiple action in the same .py? 

### ISSUE THREAD

**Issue by DraxFTW** [rhiever/TwitterFollowBot#119]
for an example like this;

my_bot = TwitterBot()
my_bot.auto_follow("@ShoutGamers")

from TwitterFollowBot import TwitterBot

my_bot = TwitterBot()
my_bot.auto_fav("@ShoutGamers", count=1000)

will this both work? or just the top one.

**Comment by rhiever** [comment]
Yes, you can do that. You don't even need to create a new instance of the TwitterBot; you can just follow one function call with another:

```python
from TwitterFollowBot import TwitterBot
my_bot = TwitterBot()
my_bot.auto_follow("@ShoutGamers")
my_bot.auto_fav("@ShoutGamers", count=1000)
```

**Comment by ghost** [comment]
Sorry for hijacking this post but what if you wanted to run 3 accounts to follow 1000 people each day. Could i put them all under one .py?

**Comment by rhiever** [comment]
Yes, if you create different `TwitterBot` instances pointing to your separate config files.

**Comment by ghost** [comment]
Thank you ive done that just worried that within the 24 hours each day the 3 accounts will not follow 1000 people each if that makes sense. If i put them all in one py will they all be following accounts at the same time or would it be one account following till 1000 before the next account starts following

**Comment by mablono5** [comment]
You are absolutely right, the count will complete to 1000 before moving on to the next job.
Setting the max time to 60 seconds means 1000 minutes. Around 16 hours per job. 48 hours to run through your script.
